smac/smbo/smbo.py

```diff
@@ -204,13 +204,14 @@ def choose_next(self, X, Y,
             else:
                 num_configurations_by_local_search = 10
 
-        # initial SLS by incumbent +
-        # best configuration from next_configs_by_random_search_sorted
+        # initiate local search with best configurations from previous runs
+        configs_previous_runs = self.runhistory.get_all_configs()
+        configs_previous_runs_sorted = self._get_acq_values_sorted(configs_previous_runs)
```

Comment:
The idea here is to reproduce the behavior of the original SMAC implementation. I can see your point that it might be good to also start the local search from random points, but we should first get the original implementation up and tested.